addons/account/models/account_analytic_plan.py

```diff
@@ -29,15 +29,35 @@ class AccountAnalyticApplicability(models.Model):
         compute='_compute_display_account_prefix',
         help='Defines if the field account prefix should be displayed'
     )
+    account_prefix_placeholder = fields.Char(compute='_compute_prefix_placeholder')
+
+    @api.depends('account_prefix', 'business_domain')
+    def _compute_prefix_placeholder(self):
+        for applicability in self:
+            account = self.env['account.account'].search(
+                [('account_type', '=', 'expense' if applicability.business_domain == 'bill' else 'income')],
+                limit=1
+            )
```

Comment:
Note that a search in a loop is an anti pattern because of performances.
This field is only in a form view for now, but if it were to be in the list view, we would have to cache the value of the search depending on the input domain

Reply:
It is actually always in a list view (embedded in the form view)